cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -28,9 +28,11 @@ After you create the pipelines for your application, you can view and visually i
 
 include::modules/op-constructing-pipelines-using-pipeline-builder.adoc[leveloffset=+1]
 
-== Creating applications with OpenShift Pipelines
+== Creating OpenShift Pipelines along with applications
 
-To create pipelines along with applications, use the *From Git* option in the *Add* view of the *Developer* perspective. For more information, see xref:../../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-importing-codebase-from-git-to-create-application_odc-creating-applications-using-developer-perspective[Creating applications using the Developer perspective].
+To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You can view all of your available pipelines and select the pipelines you want to use to create applications, while importing your code or deploying an image. You will also find a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. If an administrator has opted out of Tekton Hub integration, you will be able to see tasks from Tekton Hub which are supported by the cluster. You can also check if a webhook URL exists for a generated pipeline.
```

Comment:
Question: On 2nd thought, I am not sure about the order of the events here that I get to see. What is the 1st thing that I get to see? This is my understanding: A default pipeline that is selected for the application? I understand that despite this default pipeline, I as a user am able to view all the available pipelines and select the one I want to create applications. Correct? 

If so, please rewrite it to suit this order. Something like this:

```suggestion
To create pipelines along with applications, use the *From Git* option in the *Add+* view of the *Developer* perspective. You see a default pipeline selected and can click *Show Pipeline Visualization* to see the pipeline for the application. You can view all of your available pipelines and select the pipelines you want to use to create applications while importing your code or deploying an image. If an administrator has opted out of Tekton Hub integration, you can see tasks from Tekton Hub which are supported by the cluster. You can also check whether a webhook URL exists for a generated pipeline.
```